chain/evm/compass.go

```diff
@@ -617,3 +840,111 @@ func (c compass) callCompass(
 	}
 	return c.evm.ExecuteSmartContract(ctx, c.chainID, *c.compassAbi, c.smartContractAddr, useMevRelay, method, arguments)
 }
+
+func (t compass) gravityRelayBatches(ctx context.Context, batches []chain.GravityBatchWithSignatures) error {
+	var gErr whoops.Group
+	logger := log.WithField("chainReferenceID", t.ChainReferenceID)
+	for _, batch := range batches {
+		logger = logger.WithField("batch-nonce", batch.BatchNonce)
+
+		if ctx.Err() != nil {
+			logger.Debug("exiting processing batch context")
+			break
+		}
+
+		var processingErr error
+		var tx *ethtypes.Transaction
+		logger := logger.WithFields(log.Fields{
+			"batch-nonce": batch.BatchNonce,
+		})
+		logger.Debug("relaying")
+
+		tx, processingErr = t.gravityRelayBatch(ctx, batch)
+
+		processingErr = whoops.Enrich(
+			processingErr,
+			FieldMessageID.Val(batch.BatchNonce),
+		)
+
+		switch {
+		case processingErr == nil:
+			if tx != nil {
+				logger.Debug("sending claim")
+				// TODO : Claim
+				//if err := t.paloma.SetPublicAccessData(ctx, t.ChainReferenceID, batch.BatchNonce, tx.Hash().Bytes()); err != nil {
+				//	gErr.Add(err)
+				//	return gErr
+				//}
```

Comment:
Is this alright to keep like this?

Reply:
This isn't needed anymore.  Claiming happens elsewhere.  I'll update the code